src/man/authselect.8.txt.in.in

```diff
@@ -81,6 +81,48 @@ To print help for selected command run *authselect COMMAND --help*.
 *disable-feature* feature::
     Disable feature in currently selected profile.
 
+*create-profile* NAME [--custom,-c|--vendor,-v] [options]::
+    Create a new custom profile named _NAME_. The profile can be based on an 
+    existing profile in which case the new profile templates are either copied
+    from the base profile or a symbolic links to these files are created if
+    such option is selected.
+    
+    *--vendor,-v*:::
+        The new profile is a vendor profile instead of a custom profile. See
+        _authselect-profiles(5)_ for more information on profile types.
+    
+    *--base-on=BASE-ID, -b=BASE-ID*:::
+        The new profile will be based on a profile named _BASE-ID_. The base
+        profile location is determined with these steps:
+        . If _BASE-ID_ starts with prefix _custom/_ it is a custom profile.
+        . Try if _BASE-ID_ is found in vendor profiles.
+        . Try if _BASE-ID_ is found in default profiles.
+        . Return an error.
+    
+    *--base-on-default*:::
+        The base profile is a default profile even if it is found also within
+        vendor profiles.
```

Comment:
I'm sorry, but I donn't know how is this option useful. Can you give me an example?

Reply:
As you know, we have `default`, `vendor` and `custom` profile types. The vendor type has the ability to overwrite default type. That means if you have sssd default profile, you can create sssd vendor profile and change stuff. If you call `authselect select sssd` then the vendor profile is applied, you can never activate the default profile.

On the other hand, it may be useful for the users to base the new profile on the default one. So you tell the tool with this option to look in default profiles and ignore vendor profile.

Does this make sence? Do you suggest other description of the option?